specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/managedInstances.json

```diff
@@ -355,7 +355,10 @@
         "collation": {
           "description": "Collation of the managed instance.",
           "type": "string",
-          "readOnly": true
+          "x-ms-mutability": [
```

Comment:
For the moment, this swagger extension is not being used. Please reach out to @fearthecowboy for more information if you have any questions on this.